tap_salesforce/__init__.py

```diff
@@ -125,7 +125,17 @@ def do_discover(sf):
     """Describes a Salesforce instance's objects and generates a JSON schema for each field."""
     global_description = sf.describe()
 
-    objects_to_discover = {o["name"] for o in global_description["sobjects"]}
+    objects_set = {o["name"] for o in global_description["sobjects"]}
+    objects_to_discover = [
+        "Account",
+        "Contact",
+        "Lead",
+        "Opportunity",
+        "Campaign",
+        "AccountContactRelation",
+        "AccountContactRole",
+        "OpportunityContactRole",
+    ]
```

Comment:
So, I'm assuming these are the only ones we really care about then?

Reply:
yes

Reply:
How can I know that I care about an object? Who knows that?

Reply:
check out the salesforce story card on trello